CONTRIBUTING.md

```diff
@@ -53,15 +81,15 @@ Follow these instructions for contributing new recipes. The Goal headers indicat
 
 #### Only use packages in the latest package set release
 
-In other words, we do not allow anyone to override or add packages in the `packages.dhall` file for the following reasons:
```

Comment:
Converting this to [BLUF](https://en.wikipedia.org/wiki/BLUF_(communication)) style where the key info is mentioned first, with justification at the end.

I noticed Vlad having to read though a bunch of stuff to get to the important bit about how to proceed with the recipe PR.

Reply:
LGTM